infra/main.bicep

```diff
@@ -154,6 +154,12 @@ param azureSearchSku string = 'standard'
 @description('Azure AI Search Index')
 param azureSearchIndex string = 'index-${resourceToken}'
 
+@description('Azure AI Search Indexer')
```

Comment:
We have no way to turn this on in this PR - do we need to amend the allowable values for `azureSearchUseIntegratedVectorization` and then add something like 

`param azureSearchUseIntegratedVectorization = readEnvironmentVariable('AZURE_SEARCH_USE_INTEGRATED_VECTORIZATION', false)`

into the main.bicepparam

Reply:
Komal will raise another PR to do this